dbms/src/Flash/tests/gtest_func_return_type.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <AggregateFunctions/AggregateFunctionFactory.h>
+#include <AggregateFunctions/AggregateFunctionSum.h>
+#include <DataTypes/DataTypeDecimal.h>
+#include <DataTypes/DataTypeNullable.h>
+#include <TestUtils/ExecutorTestUtils.h>
+#include <TestUtils/mockExecutor.h>
+
+#include <tuple>
+#include <vector>
+
+namespace DB
+{
+namespace tests
+{
+
+class ExecutorFuncReturnTypeTestRunner : public DB::tests::ExecutorTest
```

Comment:
why use DB::tests::ExecutorTest?
it doesn't seem necessary.
looks like it should be moved to `dbms/src/Functions/tests`

Reply:
You can construct an `AggregateFunctionTest` like `ExecutorTest`.
Then move the tests to `dbms/src/AggregateFunctions/tests`.
`ExecutorTest` is only used by Executor, Aggregate Function is best to have its own test tool.

Reply:
> You can construct an `AggregateFunctionTest` like `ExecutorTest`. Then move the tests to `dbms/src/AggregateFunctions/tests`. `ExecutorTest` is only used by Executor, Aggregate Function is best to have its own test tool.

okk